scripts/validate_seasonal_sources.py

```
@@ -0,0 +1,250 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+🔬 Валидация научных источников сезонных корректировок
+Поиск и проверка рецензируемых источников для сезонных коэффициентов
+"""
+
+import requests
+import json
+import time
+from datetime import datetime
+from typing import Dict, List, Optional
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)